autogpt_platform/frontend/src/components/node-input-components.tsx

```diff
@@ -947,9 +932,9 @@ const NodeKeyValueInput: FC<{
       <div>
         {keyValuePairs.map(({ key, value }, index) => (
           /*
-          The `index` is used as a DOM key instead of the actual `key`
-          because the `key` can change with each input, causing the input to lose focus.
-          */
+            The `index` is used as a DOM key instead of the actual `key`
+            because the `key` can change with each input, causing the input to lose focus.
+            */
```

Comment:
this looks weird to me

Reply:
could be `//` instead of `/*`, but not double indented like this

Reply:
Out of scope